dir.props

```diff
@@ -16,6 +16,14 @@
 
   <Import Project="$(MSBuildThisFileDirectory)RepoDirectories.props" />
 
+  <PropertyGroup>
+    <!--
+      This name is used to create a GIT repository URL https://github.com/dotnet/$(GitHubRepositoryName) used to find source code for debugging
+      It is also used to name the build output manifest for orchestrated builds.
+    -->
+    <GitHubRepositoryName Condition="'$(GitHubRepositoryName)' == ''">core-setup</GitHubRepositoryName>
```

Comment:
Where do we consume this for the Publish manifest stuff?

Reply:
That's in https://github.com/dotnet/buildtools/pull/1852: `ManifestName` picks up `GitHubRepositoryName` as a default.

In CoreCLR and CoreFX, `src/publish.proj` passes `GitHubRepositoryName` into `PushToBlobFeed` directly.

Reply:
Why can't we pass in `$(PB_RepoName)` like we do for `UpdatePublishedVersions`?

Reply:
We can, I mentioned this in the description:

> CoreCLR and CoreFX already had `GitHubRepositoryName` defined, so I added it here, too. The initial reason for the property was to create GitHub raw URLs for SourceLink--if this turns out to be destabilizing in Core-Setup I can use `PB_RepoName` instead.

I guess there's the assumption that we'll want to use SourceLink in this repo at some point, and might as well do it the same way as the other repos.

Reply:
Ok - that makes sense - we should keep all 3 repos as consistent as possible/reasonable.